src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/QueueBindingProvider.cs

```diff
@@ -64,9 +64,13 @@ internal class QueueBindingProvider
             var bindingFactory = new BindingFactory(nameResolver, converterManager);
 
             var bindAsyncCollector = bindingFactory.BindToAsyncCollector<QueueAttribute, IStorageQueueMessage>(BuildFromQueueAttribute, ToWriteParameterDescriptorForCollector, CollectAttributeInfo);
```

Comment:
Yes - but let's do that in a separate change. 

---
In reply to: [97888844](https://github.com/Azure/azure-webjobs-sdk/pull/967#discussion_r97888844) [](ancestors = 97888844)